src/plugin.yml

```diff
@@ -1,8 +1,11 @@
-name: CommandTimer
-version: 3.0.2.1
-description: This plugin allows you to schedule commands that will be executed
 main: me.playbosswar.com.Main
+name: "${project.name}"
```

Comment:
I suppose this is coming from Maven? Never seen it like this, but I'm not an expert in Maven either.

Reply:
When you build the plugin with maven it will replace that placeholder with the variable defined in the pom.xml file